lndr-backend/db/create_tables.sql

```diff
@@ -1,14 +1,15 @@
 CREATE TABLE pending_credits (
-    hash        CHAR(64) PRIMARY KEY,
-    submitter   CHAR(40),
-    nonce       NUMERIC(78),
-    creditor    CHAR(40),
-    debtor      CHAR(40),
-    amount      NUMERIC(78),
-    memo        CHAR(32),
-    signature   CHAR(130),
-    ucac        CHAR(40),
-    created_at  TIMESTAMP DEFAULT now()
+    hash                CHAR(64) PRIMARY KEY,
+    submitter           CHAR(40),
+    nonce               NUMERIC(78),
+    creditor            CHAR(40),
+    debtor              CHAR(40),
+    amount              NUMERIC(78),
+    memo                CHAR(32),
+    signature           CHAR(130),
+    ucac                CHAR(40),
+    settlement_currency CHAR(3),
```

Comment:
⚡This app doesn't have a formal mechanism for schema migrations, yet. Although it wouldn't be incorporated in an automated manner, let's add a migration file to this directory with the appropriate `ALTER TABLE` clause to add this column.

Reply:
☝️ Rails does migrations really well. To borrow from its conventions: migrations should be in a **migrations** subdirectory and should have a filename like **yyyy-MM-ddTHH.mm.ssZ_action_table.sql** (i.e: file-safe ISO-8601 timestamp followed by a snake_case summary of the migration's behavior, like "add_settlement_currency_to_pending_credits").